src/librustc/ty/sty.rs

```diff
@@ -930,11 +933,15 @@ impl<'a, 'gcx, 'tcx> TyS<'tcx> {
         }
     }
 
-    pub fn is_empty(&self, _cx: TyCtxt) -> bool {
+    pub fn is_empty(&self, cx: TyCtxt) -> bool {
```

Comment:
I think that this function should be renamed -- it sounds very much like it is checking for `TyEmpty`, when in fact it does more (not sure if this happens in a later commit)